api-lib/handleRefundGiveMsg.ts

```diff
@@ -0,0 +1,32 @@
+import assert from 'assert';
```

Comment:
I'm not expert with `assert` in this environment, but usually in other frameworks it's only for things you really don't expect to happen. I bring it up not exactly because It's used here but because I've seen it in other project code. i'm not sure if there is a better pattern here, especially wrt HTTP errors. In this case, it does seem really unexpected, but perhaps we could wrap it with something that that could take an http error, e.g `assertWithStatus(users_by_pk, 500)` (or 404).

Reply:
This is a succinct way to cause a throw. It's much easier than using an if statement, then generating an error manually.